permissions/plugin-sematext.yml

```
@@ -0,0 +1,7 @@
+---
+name: "sematext"
+github: "jenkinsci/sematext-plugin"
+paths:
+- "com/sematext/jenkins/plugins"
```

```suggestion
- "io/jenkins/plugins/sematext"
```

See: https://github.com/jenkinsci/sematext-plugin/blob/master/pom.xml#L14